tests/Collector/RouterCollectorTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Tests\Collector;
+
+use Yiisoft\Di\Container;
+use Yiisoft\Router\Group;
+use Yiisoft\Router\Route;
+use Yiisoft\Router\RouteCollectorInterface;
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+use Yiisoft\Yii\Debug\Collector\RouterCollector;
+
+final class RouterCollectorTest extends CollectorTestCase
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Yiisoft\Router\RouteCollectorInterface
+     */
+    private $routeCollector;
```

Comment:
Do we need it shared in every test?

Reply:
That's a hack to save the state of `$routeCollector`. Yes, we do.